doc/rtd/development/index.rst

```diff
@@ -1,83 +1,35 @@
 .. _contributing:
 
-How to contribute to cloud-init
-*******************************
-
-Thank you for wanting to help us improve cloud-init! There are a variety of
-ways you can contribute to this project, including no-code and low-code
-options. This documentation will help orient you with our processes.
-
-Please make sure that you read this guide before starting to contribute. It
-contains all the details you need to know to give your contribution the best
-chance of being accepted.
-
-Cloud-init is hosted and managed on `GitHub`_. If you're not
-familiar with how GitHub works, their
-`quickstart documentation`_
-provides an excellent introduction to all the tools and processes you'll need
-to know.
-
-.. _contributing-prerequisites:
-
-Prerequisites
-=============
-
-Before you can begin, you will need to:
-
-* Read and agree to abide by our `Code of Conduct`_.
-
-* Sign the Canonical `contributor license agreement <CLA_>`_. This grants us your
-  permission to use your contributions in the project.
-
-* Create (or have) a GitHub account. We will refer to your GitHub username as
-  ``GH_USER``.
-
-Getting help
-============
-
-We use Matrix and have a dedicated `#cloud-init` room where you can contact
-us for help and guidance. This link will take you directly to our
-`room on Matrix <Matrix_>`_.
-
-Getting started
-===============
-
 .. toctree::
     :maxdepth: 1
+    :hidden:
 
-    find_issues.rst
     first_PR.rst
-    code_review.rst
-
-Contribute
-==========
+    find_issues.rst
 
-Pull request checklist
-----------------------
+How to contribute to cloud-init
+*******************************
 
-Before any pull request can be accepted, remember to do the following:
+Thank you for your interest in improving cloud-init! There are a variety of
+ways you can contribute to this project, including no-code and low-code
+options. This documentation will help orient you to the project.
 
-* Add or update any :ref:`unit tests<testing>` accordingly.
-* Add or update any :ref:`integration_tests` (if applicable).
-* Format code (using ``black`` and ``isort``) with ``tox -e do_format``.
-* Ensure unit tests and/or linting checks pass using ``tox``.
-* Submit a PR against the ``main`` branch of the cloud-init repository.
+Make sure to read the `first PR guide<first-PR>` before starting
```

Comment:
1. We've redacted this index to something that only drives contributors to code contribution via first-PR, and omits bug filing, docs or testing contributions.

Can we retain cross-links to the other contribution scopes otherwise this index page feels spartan and leaves the other nested menu items form the left-side navigation without much context.


* Contribute code
* Contribute to documentation
* Contribute testing
* Contribute bugs or feature requests



2. Also, let's fix the typo resulting in bogus sphinx anchor
```suggestion
Make sure to read the `first PR guide<first_PR>` before starting
```